packages/web-components/examples/codesandbox/components/masthead-l1/cdn.html

```diff
@@ -16,15 +16,25 @@
   <link rel="stylesheet" href="src/index.scss" />
   <style type="text/css">
     /* Suppress custom element until styles are loaded */
-    cds-masthead-composite:not(:defined) {
+    c4d-masthead-container:not(:defined) {
       display: none;
     }
   </style>
-  <script type="module" src="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/tag/v1/latest/masthead.min.js"></script>
-  <script type="module" src="src/index-cdn.js"></script>
+  <script src="src/index-cdn.js" defer></script>
+  <script src="src/ddo.js"></script>
+
+  <!-- IBM Tag Management and Site Analytics -->
+  <script src="//1.www.s81c.com/common/stats/ibm-common.js" async="async" defer></script>
+  <!-- IBM CMaaS -->
+  <script src="//www.ibm.com/common/digitaladvisor/cm-app/latest/cm-app.min.js" defer></script>
+
+  <!-- CDN bundle -->
+  <script type="module" src="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/tag/v2/latest/masthead.min.js"></script>
```

Comment:
Followed the v1 src pattern here, but could use verification that this path is correct.

Reply:
Tested recently and this appears to be correct.